### PR TITLE
SD-4070 HTTP Push - send published package and its items in a single request in specified format

### PR DESCRIPTION
**Pull request by mugurrus**
Retrieve embedded items in packages and create corresponding
records.

### CODE REVIEW

File: content_api/publish/service.py [hunk 1]
```diff
@@ -30,10 +30,22 @@ def create(self, docs, **kwargs):
...
+            self.update(_id, doc, original)
+            return _id
+        else:
+            return super().create([doc], **kwargs)
+
+    def _process_associations(self, doc):
+        if 'associations' in doc:
+            for assoc_list in doc['associations'].values():
+                for assoc in assoc_list:
+                    if len(assoc) > 2:
```
Comment (akintolga) [inline content_api/publish/service.py:51]: Is '2' used to ignore the "sidebars"? Imo, at least a comment in the code would be useful for the purpose.

Comment (mugurrus) [inline content_api/publish/service.py:51]: Association dict contains "_id" and "type" fields. When the item was embedded it contains more than 2 fields.

Comment (akintolga) [inline content_api/publish/service.py:51]: :+1:  Maybe a line comment would be useful.

**Comment by akintolga** [comment]
:+1:

File: content_api/publish/service.py (hunk 1 above)
Comment (mugurrus) [inline content_api/publish/service.py:51]: Will do

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**